### PR TITLE
Add Angular framework

### PR DESCRIPTION
**Pull request by Alorel**
Added link to Angular 4.x/5.x framework implementation of the driver.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,6 +201,7 @@ models in your framework so you can store data offline with localForage. We
 have drivers for the following frameworks:
 
 * [AngularJS](https://github.com/ocombe/angular-localForage)
+* [Angular](https://www.npmjs.com/package/ngforage)
```
Comment (tofumatt) [inline README.md]: I think this should say `AngularJS 4+` or something like that, to differentiate.

Comment (Alorel) [reply README.md]: @tofumatt b3013f901907b1b92efea8f9eafa6faa7d6ce756 :slightly_smiling_face:

Comment (Alorel) [reply README.md]: I stuck to Angular vs AngularJS as AngularJS is widely known to refer to 1.x whilst Angular is known to refer to 2.x+

Comment (tofumatt) [reply README.md]: Ah okay, cool. I wasn't sure. 😄

**Pull request merged**